R/validate.R

```diff
@@ -64,8 +64,6 @@ dm_validate <- function(x) {
     select(table = ref_table, column = ref_column) %>%
     check_colnames(dm_col_names, "Parent key")
 
-  stopifnot(lengths(def$pks) %in% 0:1)
-
```

Comment:
This should be replaced with a correct test, not removed.

Reply:
This is still an issue. Do we need `map_int(vctrs::vec_size)` ?

Reply:
We want to test here if we have maximally 1 PK for each table, right?
I am gonna go with your suggestion here, `all(map_int(def$pks, vctrs::vec_size) %in% 0:1)` seems to work.